config/secrets.yml

```diff
@@ -246,9 +269,20 @@ production:
     security_token: <%= ENV['SALESFORCE_SECURITY_TOKEN'] %>
     consumer_key: <%= ENV['SALESFORCE_CONSUMER_KEY'] %>
     consumer_secret: <%= ENV['SALESFORCE_CONSUMER_SECRET'] %>
+    push_lead_enabled: <%= ENV['SALESFORCE_PUSH_LEAD_ENABLED'] || true %>
+    sync_accounts_enabled: <%= ENV['SALESFORCE_SYNC_ACCOUNTS_TO_SALESFORCE'] || true %>
+    sync_contacts_enabled: <%= ENV['SALESFORCE_SYNC_CONTACT_ENABLED'] || true %>
+    number_of_days_contacts_modified: <%= ENV['SALESFORCE_NUMBER_OF_DAYS_MODIFIED'] || 7 %>
```

Comment:
Might wanna use `.to_i` or similar to cast the string to an integer (or use '7' as the default value instead and handle as a string).

Reply:
had to revert this back - it is actually being handled as a string in the code already

Reply:
I would change it to "7" just to make it clear it works for strings like the ENV var and to make sure both sides of the `||` are the same type. The YAML template will substitute the string in and it'll look like:

number_of_days_contacts_modified: 7

so then that gets automatically converted to a number...